content/hamel-dev-creating-a-llm-as-a-judge-that-drives-business-results-hamel-s-blog.md

```
@@ -0,0 +1,741 @@
+### Generating Data [Anchor](https://hamel.dev/blog/posts/llm-judge/\#generating-data)
+
+To build your dataset, you can:
+
+- **Use Existing Data**: Sample real user interactions or behaviors from your AI system.
+- **Generate Synthetic Data**: Use LLMs to create realistic user inputs covering various features, scenarios, and personas.
+
+Often, you’ll do a combination of both to ensure comprehensive coverage. Synthetic data is not as good as real data, but it’s a good starting point. Also, we are only using LLMs to generate the user inputs, not the LLM responses or internal system behavior.
+
+Regardless of whether you use existing data or synthetic data, you want good coverage across the dimensions you’ve defined.
+
+**Incorporating System Information**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)